### DISCUSSION TITLE
Pop up address bar ?

### DISCUSSION THREAD

**Discussion by SheepDomination** [zen-browser/desktop#2039] [Ideas]
Instead of having to go to the top title bar for the address bar. How about a popup address bar using a keyboard short-cut ?

**Comment by ficd0** [comment]
You can bind a shortcut, `Ctrl + L` by default, to focus the address bar & highlight the text currently in it. I do get what you mean though; I think there may already be some Zen Mods that have this behavior.

**Reply by SheepDomination** [reply]
It would be nice as a browser option, instead of as a mod; although which mods ?
